config/routes.rb

```diff
@@ -7,8 +7,13 @@
   
   namespace :api do
     namespace :v1 do
+      use_doorkeeper do
+        controllers tokens: 'tokens'
+        skip_controllers :authorizations, :applications, :authorized_applications, :token_info, :tokens
+      end
       resources :keywords, only: :index
       resources :private_items, only: :index
+      post '/login', to: 'tokens#create'
```

Comment:
As usual, unless there is a specific need from the requirements or any technical constraints, we recomend sticking with the Rails standard.
This applies with routes: `tokens` is a resource, so we can use the `resources` short hand:
```suggestion
      resources :tokens, only: [:create]
```

Then we will have the route `POST /tokens` created (instead of `POST /login`).

Reply:
@phongvhd93 You might want to do the same on your PR 🙏